packages/dashboard-v2/src/pages/settings/developer-settings.js

```diff
@@ -31,53 +32,60 @@ const APIKeysPage = () => {
-        <title>API Keys</title>
+        <title>Developer settings</title>
       </Metadata>
       <div className="flex flex-col xl:flex-row">
-        <div className="flex flex-col gap-10 lg:shrink-0 lg:max-w-[576px] xl:max-w-[524px]">
+        <div className="flex flex-col gap-10 lg:shrink-0 lg:max-w-[576px] xl:max-w-[524px] leading-relaxed">
           <div>
-            <h4>API Keys</h4>
-            <p className="leading-relaxed">There are two types of API keys that you can generate for your account.</p>
-            <p>Make sure to use the appropriate type.</p>
+            <h4>Developer settings</h4>
+            <p>API keys allow developers and applications to extend the functionality of your portal account.</p>
+            <p>Skynet uses two types of API keys, explained below.</p>
           </div>
 
           <hr />
 
           <section className="flex flex-col gap-2">
-            <h5>Public keys</h5>
-            <p className="text-palette-500">
-              Public keys provide read access to a selected list of skylinks. You can share them publicly.
+            <h5>Sponsor keys</h5>
+            <div className="text-palette-500"></div>
+            <p>
+              Sponsor keys allow users without an account on this portal to download skylinks covered by the API key.
             </p>
-
+            <p>
+              Learn more about sponsoring content with Sponsor API Keys{" "}
+              <HighlightedLink as="a" href="#">
+                here
+              </HighlightedLink>
+              .
+            </p>{" "}
+            {/* TODO: missing documentation link */}
             <div className="mt-4">
               <AddPublicAPIKeyForm ref={publicFormRef} onSuccess={refreshState} />
             </div>
-
             {error ? (
               <Alert $variant="error" className="mt-4">
-                An error occurred while loading your API keys. Please try again later.
+                An error occurred while loading your sponsor keys. Please try again later.
               </Alert>
             ) : (
               <div className="mt-4">
-                {publicKeys?.length > 0 ? (
-                  <APIKeyList title="Your public keys" keys={publicKeys} reloadKeys={() => refreshState(true)} />
+                {sponsorKeys?.length > 0 ? (
+                  <APIKeyList title="Your public keys" keys={sponsorKeys} reloadKeys={() => refreshState(true)} />
                 ) : (
-                  <Alert $variant="info">No public API keys found.</Alert>
+                  <Alert $variant="info">No sponsor keys found.</Alert>
                 )}
               </div>
             )}
           </section>
+
           <hr />
 
           <section className="flex flex-col gap-2">
-            <h5>General keys</h5>
+            <h5>API keys</h5>
             <p className="text-palette-500">
               These keys provide full access to <b>Accounts</b> service and are equivalent to using a JWT token.
```

Comment:
REQ: This is no longer true. API keys allow uploading and downloading skyfiles, as well as reading and writing to the registry. Nothing more.